Add Ife.json for Web3 Developer Portfolio

Juncann01 opened the pull request:
<!--
YOU MUST FILL OUT THIS TEMPLATE ENTIRELY FOR YOUR PR TO BE ACCEPTED, NONE OF IT IS OPTIONAL UNLESS SPECIFIED.
-->

# Requirements
<!-- Your domain MUST pass ALL the requirements below, otherwise it WILL BE DENIED. -->

<!-- Change each checkbox to [x] (all lowercase, with no spaces between the brackets) to mark it as completed. -->

- [x] I have **read** and **agreed** to the [Terms of Service](https://is-a.dev/terms). <!-- Your request MUST follow the TOS to be approved. -->
- [x] My file is following the [domain structure](https://docs.is-a.dev/domain-structure/).
- [x] My website is **reachable** and **completed**. <!-- We do not permit simple "Hello, world!" or simply copied/mostly blank templated websites. -->
- [x] My website is **software development** related. <!-- Only your root subdomain needs to meet this requirement. -->
- [x] My website is not for commercial use. <!-- Your website's purpose should not be to generate any form of revenue. -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g. X/Twitter or Discord) for contact. -->
- [x] I have provided a preview of my website below. <!-- This step is required for your domain to be approved. -->

# Website Preview
<!-- Provide a link or screenshot of your website below. -->

https://ifeweb3.pxxl.click/

Juncann01 committed:
Add Ife.json for Web3 Developer Portfolio

STICKnoLOGIC commented on Ife.json:
- move this file inside of domain folder
- filename must be in lowercase

STICKnoLOGIC commented on Ife.json:
```diff
@@ -0,0 +1,9 @@
+    "email": "ifeo55394@gmail.com"
+  },
+  "record": {
+    "CNAME": "ifeweb3.pxxl.click/"
```
```suggestion
    "CNAME": "ifeweb3.pxxl.click"
```

Juncann01 committed:
Merge branch 'main' into main

Juncann01 committed:
Rename Ife.json to domains/ifeweb3.pxxl.click/ife.json

Thanks for the review — I've moved the JSON into domains/ifeweb3.pxx1.click/lfe.json, lowercased the filename, added my GitHub username, and changed record -> records with the CNAME entry. @maintainer Could you please approve the workflow so CI can run? 🙏

STICKnoLOGIC commented on domains/ifeweb3.pxxl.click/ife.json:
move this file inside of domain folder

STICKnoLOGIC reviewed (commented):
Closing PR:
- Lack of response for at most 2 days since the first comment by maintainer/helper
- resolve issues below
